src/main/java/uk/gov/ons/ssdc/supporttool/utility/JobTypeHelper.java

```diff
@@ -31,37 +40,123 @@
         jobTypeSettings.setFileLoadPermission(UserGroupAuthorisedActivityType.LOAD_BULK_REFUSAL);
-        jobTypeSettings.setFileViewProgressPersmission(
+        jobTypeSettings.setFileViewProgressPermission(
             UserGroupAuthorisedActivityType.VIEW_BULK_REFUSAL_PROGRESS);
         return jobTypeSettings;
 
+      case BULK_INVALID:
+        jobTypeSettings.setTransformer(BULK_INVALID_TRANSFORMER);
+        jobTypeSettings.setColumnValidators(getBulkInvalidCaseValidationRules(collectionExercise));
+        jobTypeSettings.setTopic(
+            toProjectTopicName(invalidCaseTopic, sharedPubsubProject).toString());
+        jobTypeSettings.setFileLoadPermission(UserGroupAuthorisedActivityType.LOAD_BULK_INVALID);
+        jobTypeSettings.setFileViewProgressPermission(
+            UserGroupAuthorisedActivityType.VIEW_BULK_INVALID_PROGRESS);
+        return jobTypeSettings;
+
+      case BULK_UPDATE_SAMPLE:
+        jobTypeSettings.setTransformer(BULK_SAMPLE_UPDATE_TRANSFORMER);
+        jobTypeSettings.setColumnValidators(
+            getBulkSampleValidçationRulesHeaderRowOnly(collectionExercise));
+        jobTypeSettings.setSampleAndSensitiveDataColumnMaps(
+            collectionExercise.getSurvey().getSampleValidationRules(), collectionExercise);
+        jobTypeSettings.setTopic(
+            toProjectTopicName(updateSampleTopic, sharedPubsubProject).toString());
+        jobTypeSettings.setFileLoadPermission(
+            UserGroupAuthorisedActivityType.LOAD_BULK_UPDATE_SAMPLE);
+        jobTypeSettings.setFileViewProgressPermission(
+            UserGroupAuthorisedActivityType.VIEW_BULK_UPDATE_SAMPLE_PROGRESS);
+
+        return jobTypeSettings;
+
+      case BULK_UPDATE_SAMPLE_SENSITIVE:
+        jobTypeSettings.setTransformer(BULK_SENSITIVE_UPDATE_TRANSFORMER);
+        jobTypeSettings.setColumnValidators(
+            getBulkSampleValidçationRulesHeaderRowOnly(collectionExercise));
+        jobTypeSettings.setSampleAndSensitiveDataColumnMaps(
+            collectionExercise.getSurvey().getSampleValidationRules(), collectionExercise);
+        jobTypeSettings.setTopic(
+            toProjectTopicName(updateSensitiveSampleTopic, sharedPubsubProject).toString());
+        jobTypeSettings.setFileLoadPermission(
+            UserGroupAuthorisedActivityType.LOAD_BULK_UPDATE_SAMPLE_SENSITIVE);
+        jobTypeSettings.setFileViewProgressPermission(
+            UserGroupAuthorisedActivityType.VIEW_BULK_UPDATE_SAMPLE_SENSITIVE_PROGRESS);
+
+        return jobTypeSettings;
+
       default:
         // This code should be unreachable, providing we have a case for every JobType
         throw new RuntimeException(
             String.format("In getJobTypeSettings the jobType %s wasn't matched", jobType));
     }
   }
 
-  private ColumnValidator[] getBulkProcessorValidationRules() {
-    Rule[] caseExistsRules = {new CaseExistsRule()};
+  private ColumnValidator[] getBulkSampleValidçationRulesHeaderRowOnly(
```

Comment:
Spelling error on method name: `Validçation`

Reply:
Cheers, I removed some of those alrready, wouldn't even know how to add a 'ç'